src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star stars--0">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star stars__star--margin--0"></div>
+    </div>
+
+    <div class="star stars--1">
+      <div class="stars__star stars__star--is-active"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
you don't need this modifier